far/filelist.cpp

```diff
@@ -4879,10 +4891,14 @@ bool FileList::ApplyCommand()
 				break;
 
 			string strConvertedCommand = strCommand;
+      string strDescription;
 			list_names ListNames;
 			bool PreserveLFN = false;
 
-			if (SubstFileName(strConvertedCommand, subst_context(i.FileName, i.AlternateFileName()), &ListNames, &PreserveLFN) && !strConvertedCommand.empty())
+      if (i.DizText)
+        strDescription = i.DizText;
+
+			if (SubstFileName(strConvertedCommand, subst_context(i.FileName, i.AlternateFileName(), strDescription), &ListNames, &PreserveLFN) && !strConvertedCommand.empty())
```

Comment:
You don't need a string (and additional allocation) here at all, `subst_context(..., NullToEmpty(i.DizText))` should do.

Reply:
confirmed. will likely be resolved by one of the earlier issues.

Reply:
changed in 5b457b1.